### PR TITLE
Remove PRs to Endless Sky Assets and High DPI repos from the PR template

### PR DESCRIPTION
**Pull request by TheGiraffe3**
I believe the title sufficiently explains this.

### CODE REVIEW

**Comment by Zitchas** [comment]
We should probably add a comment about including the X2 versions in the or alongside normal images, and just attach a zip of the assets to the original post in GitHub. That should be sufficient access for now.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -33,9 +33,7 @@ This save file can be used to test these changes:
 
 ## Artwork Checklist
 (If any artwork was added or changed by this PR, the following must be provided.)
- - [ ] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
- - [ ] I created a PR to the [endless-sky-assets repo](https://github.com/endless-sky/endless-sky-assets) with the necessary image, blend, and texture assets: {{insert PR link}}
- - [ ] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: {{insert PR link}}
+ - [ ] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified.
```
Comment (TheGiraffe3) [inline .github/pull_request_template.md:36]:
```suggestion
 - [ ] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified.

Please upload a zipped file with @2x assets and HighDPI images to the main body of your pull request.
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -33,9 +33,9 @@ This save file can be used to test these changes:
...
```
Comment (Zitchas) [inline .github/pull_request_template.md:36]:
```suggestion
 - [ ] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified.
 - [ ] Include the '@2x' version of any added images alongside the regular image.
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -33,9 +33,9 @@ This save file can be used to test these changes:
...
+
+Please upload a zipped file with @2x assets and HighDPI images to the main body of your pull request.
```
Comment (Zitchas) [inline .github/pull_request_template.md]:
```suggestion
- [ ] Please upload a zipped file with the necessary image, blend, and texture assets to the main body of your pull request.
```

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -33,9 +33,10 @@ This save file can be used to test these changes:
 
 ## Artwork Checklist
 (If any artwork was added or changed by this PR, the following must be provided.)
- - [ ] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
- - [ ] I created a PR to the [endless-sky-assets repo](https://github.com/endless-sky/endless-sky-assets) with the necessary image, blend, and texture assets: {{insert PR link}}
- - [ ] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: {{insert PR link}}
+ - [ ] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified.
+  - [ ] Include the '@2x' version of any added images alongside the regular image.
+
```
Comment (TheGiraffe3) [inline .github/pull_request_template.md]:
```suggestion
```

**Comment by Zitchas** [comment]
Looks good to me!

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -33,9 +33,9 @@ This save file can be used to test these changes:
 
 ## Artwork Checklist
 (If any artwork was added or changed by this PR, the following must be provided.)
- - [ ] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
- - [ ] I created a PR to the [endless-sky-assets repo](https://github.com/endless-sky/endless-sky-assets) with the necessary image, blend, and texture assets: {{insert PR link}}
- - [ ] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: {{insert PR link}}
+ - [ ] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified.
+  - [ ] Include the '@2x' version of any added images alongside the regular image.
```
Comment (Zitchas) [inline .github/pull_request_template.md]:
```suggestion
 - [ ] Include the '@2x' version of any added images alongside the regular image.
```

I think I had an extra space in there in the original suggestion. Sorry

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -33,9 +33,9 @@ This save file can be used to test these changes:
...
+- [ ] Please upload a zipped file with the necessary image, blend, and texture assets to the main body of your pull request.
```
Comment (Zitchas) [inline .github/pull_request_template.md]:
```suggestion
 - [ ] Please upload a zipped file with the necessary image, blend, and texture assets to the main body of your pull request.
```

And I missed a space here.

**Pull request merged**